cypress/integration/Case2_Authorization.js

```diff
@@ -5,43 +5,42 @@ import HomePage from '../PageObjects/HomePage'
 import userObject from '../fixtures/userObject.json'
 
 function checkValuesСorrectness(values, isCorrect) {
-  LoginPage.recoverWalletButton.should('not.be.enabled')
-  LoginPage.mnemonicsInput.type(values)
-  LoginPage.recoverWalletButton.click()
-  if (isCorrect) {
-    LoginPage.yayButton.click()
-    HomePage.profileAvatar.should('be.visible')
-  } else {
-    LoginPage.errorWindow.should('be.visible')
-    cy.contains('Ok').click()
-    LoginPage.mnemonicsInput.clear()
-  }
+    LoginPage.recoverWalletButton.should('not.be.enabled')
+    LoginPage.mnemonicsInput.type(values)
+    LoginPage.recoverWalletButton.click()
+    if (isCorrect) {
+        LoginPage.yayButton.click()
+        HomePage.profileAvatar.should('be.visible')
+    } else {
+        LoginPage.errorWindow.should('be.visible')
+        cy.contains('Ok').click()
+        LoginPage.mnemonicsInput.clear()
+    }
 }
 
 describe('Test case 2: Ability to do authorization', () => {
-  beforeEach(() => {
-    localStorage.setItem('TorusTestUser', JSON.stringify(userObject))
-    StartPage.open()
-    StartPage.continueOnWebButton.click()
-    StartPage.signInButton.should('be.visible')
-    StartPage.signInButton.click()
-    LoginPage.recoverFromPassPhraseLink.click()
-    LoginPage.pageHeader.should('contain', 'Recover')
-    LoginPage.mnemonicsInput.should('be.visible')
-  })
+    beforeEach(() => {
+        localStorage.setItem('TorusTestUser', JSON.stringify(userObject))
```

Comment:
i dont see how this is relevant here.
TorusTestUser is for bypassing the social login. and should be implemented in own tests like discussed